src/UIController.js

```diff
@@ -0,0 +1,58 @@
+const UIController = (() => {
+  function setCityName(name) {
+    document.querySelector('#city-name > h1').innerHTML = name.toUpperCase();
+  }
+
+  function setDescription(desc) {
+    document.querySelector('#city-description > h1').innerHTML = desc.toUpperCase();
+  }
+
+  function setCurrentTemp(temp) {
+    document.getElementById('current-temp').textContent = temp.toFixed(2);
+  }
+
+  function setCurrentFeels(temp) {
+    document.getElementById('current-feels').innerHTML = Math.round(temp);
+  }
+
+  function setCurrentWind(wind) {
+    document.getElementById('current-wind').innerHTML = Math.round(wind);
+  }
+
+  function setMaxMinTemps(minTemp, maxTemp) {
+    document.getElementById('min-temp').innerHTML = Math.floor(minTemp);
+    document.getElementById('max-temp').innerHTML = Math.ceil(maxTemp);
+  }
+
+  function setCurrentInformation(data) {
```

Comment:
- [ ] Kindly change this regular functions to arrow functions. Also, make this change for every module having a regular function expression.

Reply:
I am sorry, I really prefer the 'function blabla' pattern more. I understood that I could mix patterns.

I will keep in mind for the next projects =)